src/Whmcs/WhmcsConfig.php

```diff
@@ -22,18 +22,24 @@ class WhmcsConfig
     const SWITCH_ACCESS = 6;
     const DELETE_ACTION = 7;
     const PRE_INSTALL = 8;
+    const MEM_BILLING_ID = 9;
+    const DISK_BILLING_IDS = 10;
+    const ADDON_BILLING_IDS = 11;
 
     /**
      * The 1-based index of the last Config Option.
      *
      * @var int
      */
-    protected $countOptions = self::PRE_INSTALL;
+    protected $countOptions = self::ADDON_BILLING_IDS;
 
     const API_USER_DESC = 'This must be an administrator user with API access enabled.';
     const TICKET_DEPT_DESC = 'When provisioning fails due to low inventory, a ticket will be filed on behalf of the client in this support department.';
     const DELETE_ACTION_DESC = 'When a product is terminated, this action will occur.';
     const PRE_INSTALL_DESC = 'Billing ID of an OS Reload that will be run before each install, e.g. format-quick. Multiple can be separated by a comma.';
+    const MEM_BILLING_DESC = 'Optional preset Billing ID of the RAM. This field is overrided when the \'Memory\' Configurable Option is present. ex: mem-1|8 GB RAM';
```

Comment:
overridden* same with below 2
```suggestion
    const MEM_BILLING_DESC = 'Optional preset Billing ID of the RAM. This field is overridden when the \'Memory\' Configurable Option is present. ex: mem-1|8 GB RAM';
```